### ISSUE TITLE
Invalid entries on abuse list

### ISSUE THREAD

**Issue by pallebone** [blocklistproject/Lists#1005]
Hi there,

Pihole reports invalid entries:

  [i] Target: https://blocklistproject.github.io/Lists/abuse.txt
  [✓] Status: No changes detected
  [✓] Parsed 435166 exact domains and 0 ABP-style domains (ignored 11 non-domain entries)
      Sample of non-domain entries:
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.2oflranvbs3drvbmwvj2afac0dybr9qinmkumyfng8gnth1q4ah4fi6ecuft.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.alh5zrdnvj7gdymtzxhyea6dsbap6co6om1edtrodnv8c8rsvyg1pb4id5n0.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cfrsfp8hcpkwdzsetpo8vir0wi1t64yyfq5knbn4ckw231kifi4nz3a9st5m.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cybgzbqmymjbfcvgdi7aqceiao6d7yfvzclrsxf5whay3gcbddhg8lo81jax.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.gx86fkitocomhes0vfdqv6d1hpsyoxzulwqr8oalqooyt7xfbrwwmznue6it.jafinafara.f"

Kind regards
Peter

**Comment by pallebone** [comment]
[i] Target: https://blocklistproject.github.io/Lists/malware.txt
  [✓] Status: No changes detected
  [✓] Parsed 435272 exact domains and 0 ABP-style domains (ignored 12 non-domain entries)
      Sample of non-domain entries:
        - "protection201-account.6827-update.team.com8serv1130.webs001231cr-cm-lgin-submmit-id99-l0gin-submito9-id.ppi11781-lo0gin-loogiin-"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.2oflranvbs3drvbmwvj2afac0dybr9qinmkumyfng8gnth1q4ah4fi6ecuft.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.alh5zrdnvj7gdymtzxhyea6dsbap6co6om1edtrodnv8c8rsvyg1pb4id5n0.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cfrsfp8hcpkwdzsetpo8vir0wi1t64yyfq5knbn4ckw231kifi4nz3a9st5m.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cybgzbqmymjbfcvgdi7aqceiao6d7yfvzclrsxf5whay3gcbddhg8lo81jax.jafinafara.f"

**Comment by pallebone** [comment]
[i] Target: https://blocklistproject.github.io/Lists/phishing.txt
  [✓] Status: No changes detected
  [✓] Parsed 190229 exact domains and 0 ABP-style domains (ignored 6 non-domain entries)
      Sample of non-domain entries:
        - "blog.btdeployments.cu.m"
        - "www.barclaycard.co.uk-personal-codeb6259568de39651d7a-login.id-107sbtd9cbhsbtd5d80a13c0db1f546757jnq9j5754675780483825.exaways.c"
        - "www.https.portale-titolari-nexi-italia-online-verifica-nexi.it.infomazione.nexi-identita.conto.nexi.bp2ip.dayasinergibisnis.co.i"
        - "www.login.microsoftonline.com.common.oauth2.authorize.clientid4345a7b9.9a63.4910.a426-35363201d503.response.spreadsheetlimited.c"
        - "www.login.microsoftonline.com.common.oauth2.authorize.clientid4345a7b99a634910a42635363201d503.response.mode.form.postresponse.t"

**Comment by techroy23** [comment]
[✓] Parsed 435272 exact domains and 0 ABP-style domains (ignored 12 non-domain entries)
      Sample of non-domain entries:
        - "protection201-account.6827-update.team.com8serv1130.webs001231cr-cm-lgin-submmit-id99-l0gin-submito9-id.ppi11781-lo0gin-loogiin-"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.2oflranvbs3drvbmwvj2afac0dybr9qinmkumyfng8gnth1q4ah4fi6ecuft.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.alh5zrdnvj7gdymtzxhyea6dsbap6co6om1edtrodnv8c8rsvyg1pb4id5n0.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cfrsfp8hcpkwdzsetpo8vir0wi1t64yyfq5knbn4ckw231kifi4nz3a9st5m.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cybgzbqmymjbfcvgdi7aqceiao6d7yfvzclrsxf5whay3gcbddhg8lo81jax.jafinafara.f"

**Comment by pallebone** [comment]
List still has issues:

  [i] Target: https://blocklistproject.github.io/Lists/abuse.txt
  [✓] Status: No changes detected
  [✓] Parsed 435166 exact domains and 0 ABP-style domains (ignored 11 non-domain entries)
      Sample of non-domain entries:
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.2oflranvbs3drvbmwvj2afac0dybr9qinmkumyfng8gnth1q4ah4fi6ecuft.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.alh5zrdnvj7gdymtzxhyea6dsbap6co6om1edtrodnv8c8rsvyg1pb4id5n0.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cfrsfp8hcpkwdzsetpo8vir0wi1t64yyfq5knbn4ckw231kifi4nz3a9st5m.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cybgzbqmymjbfcvgdi7aqceiao6d7yfvzclrsxf5whay3gcbddhg8lo81jax.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.gx86fkitocomhes0vfdqv6d1hpsyoxzulwqr8oalqooyt7xfbrwwmznue6it.jafinafara.

**Comment by pallebone** [comment]
also on this list:
  [i] Target: https://blocklistproject.github.io/Lists/malware.txt
  [✓] Status: No changes detected
  [✓] Parsed 435272 exact domains and 0 ABP-style domains (ignored 12 non-domain entries)
      Sample of non-domain entries:
        - "protection201-account.6827-update.team.com8serv1130.webs001231cr-cm-lgin-submmit-id99-l0gin-submito9-id.ppi11781-lo0gin-loogiin-"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.2oflranvbs3drvbmwvj2afac0dybr9qinmkumyfng8gnth1q4ah4fi6ecuft.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.alh5zrdnvj7gdymtzxhyea6dsbap6co6om1edtrodnv8c8rsvyg1pb4id5n0.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cfrsfp8hcpkwdzsetpo8vir0wi1t64yyfq5knbn4ckw231kifi4nz3a9st5m.jafinafara.f"
        - "securelogin.paypal.it.webapps.mpp.home.autenticazione.cybgzbqmymjbfcvgdi7aqceiao6d7yfvzclrsxf5whay3gcbddhg8lo81jax.jafinafara.f"

**Comment by pallebone** [comment]
Finally this list also:

  [i] Target: https://blocklistproject.github.io/Lists/phishing.txt
  [✓] Status: No changes detected
  [✓] Parsed 190229 exact domains and 0 ABP-style domains (ignored 6 non-domain entries)
      Sample of non-domain entries:
        - "blog.btdeployments.cu.m"
        - "www.barclaycard.co.uk-personal-codeb6259568de39651d7a-login.id-107sbtd9cbhsbtd5d80a13c0db1f546757jnq9j5754675780483825.exaways.c"
        - "www.https.portale-titolari-nexi-italia-online-verifica-nexi.it.infomazione.nexi-identita.conto.nexi.bp2ip.dayasinergibisnis.co.i"
        - "www.login.microsoftonline.com.common.oauth2.authorize.clientid4345a7b9.9a63.4910.a426-35363201d503.response.spreadsheetlimited.c"
        - "www.login.microsoftonline.com.common.oauth2.authorize.clientid4345a7b99a634910a42635363201d503.response.mode.form.postresponse.t"

**Comment by gap579137** [comment]
Triage applied: status:triaged and bot labels added.

Commit SHA: `842705b`

This issue was automatically processed by Hermes.

**Comment by AxiomOperator** [comment]
This issue is being closed as part of a repository cleanup and issue triage refresh.

We are starting with a clean slate to better manage the blocklist project going forward. If this issue is still relevant, please feel free to open a new issue with:
- Updated information
- Current evidence/examples
- Clear description of the request

Thank you for your contribution to the Block List Project!
